src/intops/impl/inlineasm/arm64.nim

```diff
@@ -0,0 +1,62 @@
+## Arithmetic operations for integers implemented in ARM64 GCC/Clang inline Assembly.
+
+import ../../consts
+
+when cpuArm64 and compilerGccCompatible and canUseInlineAsm:
```

Comment:
Kinda wonder how to check this path (and others) is taken given the lack of coverage tooling. But ig for this lib the benchmark will show the perf diff when testing pure VS native.

Reply:
I also thought about that. Basically, you can determine that by examining the dispatcher. We could also inject debug logging that would be omitted in release builds.

But you are right, in the benchmarks I'm running all available implementations for every operation and if an implementation is missing, you see that in the logs: https://github.com/vacp2p/nim-intops/actions/runs/20573099021/job/59084198907#step:16:71

I want to generate the benchmark logs in HTML and include them in the docs so that it would be easy to compare things.